app/controllers/discourse_subscriptions/hooks_controller.rb

```diff
@@ -27,6 +28,38 @@ def create
       end
 
       case event[:type]
+      when "checkout.session.completed"
+        checkout_session = event[:data][:object]
+        email = checkout_session[:customer_email]
+        customer_id = checkout_session[:customer] unless checkout_session[:customer].nil?
```

Comment:
Not familiar enough with the plugin, but this seems to continue even if `customer_id` here is empty. Is that a reasonable scenario?

Reply:
Ya good catch. I'll have it return a 422 if not found.